src/charge_point.cpp

```diff
@@ -100,8 +100,42 @@ int main(int argc, char* argv[]) {
         fs::create_directories(cso_path);
     }
 
+    const fs::path csms_path = "/tmp/client/csms";
+    if (!fs::exists(csms_path)) {
+        fs::create_directories(csms_path);
```

Comment:
Shouldn't you explicitly include `ocpp/common/support_older_cpp_versions.hpp` for the `fs` namespace? It seems it is implicitly included via some other `#include`. (I don't know about EVerest's code policy on this, feel free to clarify).

Reply:
Thank you for your comments; yes, it is included over the charge_point.hpp, i have no clear opinion on that.

Reply:
@barsnick 
> implicitly included via some other #include

I'm in favour to explicitly include them. Just found the IWYU (https://include-what-you-use.org/) tool which checks this (at least it should, don't have own experience with it, yet).